### ISSUE TITLE
Workload Pass/fail criteria should not depend on expected run duration by default

### ISSUE THREAD

**Issue by smalleni** [openshift-scale/workloads#136]
Currently, a workload script can exit with a non-zero error code in two probably scenarios
1. If the workload itself fails
2. If the workload run takes more time than an expected duration set as a variable

For new users using the framework, it is not clear what these default values should be or if they work in their cluster. It would take some time and testing to come up with these expected duration values. It would make sense to make it optional for pass/fail criteria to depend on expected duration.

**Comment by smalleni** [comment]
@chaitanyaenr FYI

**Comment by chaitanyaenr** [comment]
Agreed that there should be an option to disable the duration comparison functionality with default being false to avoid any confusion like you mentioned. Also the default ( 600 seconds ) is not enough for most of the workloads to complete, so they will fail unless we tweak it to something high. Once we know the gold values, we can enable it in cases like CI to determine pass/fail.
